documentation/docs.html

```diff
@@ -491,7 +493,65 @@ <h3 class="small-title">Grid</h3>
                     style="height: 481px; width: 100%; border:0; transform: scale(1); overflow:hidden; background: #151718;"
                     sandbox="allow-scripts allow-same-origin">
                 </iframe>
+            </div>
+        </section>
+
+        <!-- Navbar -- component -->
+        <section id="navbar" class="component-container">
+            <h2 class="main-heading">Navbar</h2>
+            <p class="bold-desc">
+                Navigation bar is used to navigate through the web apps or websites. This navigation component is
+                responsive. It is a hamburger menu in medium and small devices
+            </p>
+
+            <h3 class="small-title">Navigation</h3>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quae, vero dignissimos sapiente laudantium
+                nostrum nulla ullam doloremque, perferendis quam veritatis sed dolore, similique aut amet. Aliquid harum
+                aperiam nisi iure.
+            </p>
 
+            <div class="component-preview">
+                <nav class="nav nav-boxshadow">
+                    <a href="/" class="brand">
+                        <div class="logo">
+                            <img src="../images/nav-logo.png" alt="logo">
+                        </div>
+                        <h1 class="brand-name">
+                            Fabelle-UI
+                        </h1>
+                    </a>
+                    <div class="nav-links">
+                        <a class="nav-link" href="">Home</a>
+                        <a class="nav-link" href="">Link1</a>
+                        <a class="nav-link" href="">Link2</a>
+                        <a class="nav-link" href="">Link2</a>
+                    </div>
+                    <div class="nav-search">
+                        <i class="fa fa-search"></i>
+                        <input type="text" placeholder="Type to search">
+                    </div>
+                    <div class="nav-icons">
+                        <a href="">
+                            <i class="fa fa-user"></i>
+                            <span class="nav-icon-text">Login</span>
+                        </a>
+                        <a href="">
+                            <i class="fa fa-heart" aria-hidden="true"></i>
+                            <span class="nav-icon-text">Wishlist</span>
+                        </a>
+                        <a href="">
+                            <i class="fa fa-user"></i>
+                            <span class="nav-icon-text">cart</span>
+                        </a>
+                    </div>
```

Comment:
Okay so, you can have the a tags inside unordered list. I have noticed this being used in several places and I am refactoring my code as well. This way you can have better semantic HTML code instead of using divs.

Reply:
can div and a tag be bad idea?

Reply:
Not really, but I was just saying that the above code can be written this way too.

```
<ul class='nav-icons'>
   <li> <a></a>
   </li>
</ul>

```

Reply:
ya I know but I prefer this way because I have not care about ul and li these things that's why I use and div